packages/client/src/runtime/FieldRef.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * A reference to a specific field of a specific model
+ */
+export interface FieldRef<Model, FieldType> {
+  readonly modelName: Model
+  readonly name: string
+  readonly typeName: FieldType
+  readonly isList: boolean
+}
+
+export class FieldRefImpl<Model, FieldType> implements FieldRef<Model, FieldType> {
```

Comment:
I think this belongs in core/model because anything outside core is considered "dirty", which isn't the case here :D.

Reply:
Moved the file